assets/src/realtime.d.ts

```diff
@@ -123,7 +123,7 @@ export interface Vehicle {
   operatorLogonTime: Date | null
   overloadOffset?: number
   bearing?: number
-  blockId: BlockId
+  blockId: BlockId | null
```

Comment:
question (non-blocking): In the same vein as above, I wonder if it would be helpful to split out subtypes that are LoggedOutVehicle & LoggedInVehicle so that the existing references to the `Vehicle` type still have `block_id` as a required field. Although thinking about it now, I don't think shuttles would have a blockId, either, so maybe not necessary at this point.

Reply:
I think they do have a block ID assigned (otherwise there would have been a type error with shuttle vehicles previously), but I don't think it's meaningful (I think it might be auto-assigned to a placeholder by TM or something). I think either way from Skate's perspective it should be nullable if the values for shuttles don't mean anything, and in an ideal world maybe we would just filter those values out.